<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="author" content="Niklas Pfister">
    <title></title>
    <meta name="description" content="">
    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
    <link rel="stylesheet" href="https://NiklasPfister.github.io/css/main.css">
  </head>
  <body>
    <div class="container content">
      <h1 id="bachelor-projects-in-statistics">Bachelor projects in statistics</h1>
<p><strong>Niklas Pfister, July 2023</strong></p>

<p>This page outlines thesis projects in statistics for the bachelor’s
degree in mathematics or mathematics-economy at the University of
Copenhagen. You may either pick one of the suggested topics or come up
with your own project.</p>

<p><strong>Info meeting: Monday, August 28, 3:15-4:00pm, [Room: TBA].</strong></p>

<h2 id="formalities">Formalities</h2>

<p>The thesis is written during block 1 and block 2, 2023/2024. The start
date is <strong>September 4</strong> and the thesis is handed in on <strong>January
12</strong>. There is a subsequent oral defense.</p>

<ul>
  <li>The thesis should be written in English.</li>
  <li>It’s a 15 ECTS project and you should expect to write between 30 and 45 pages.</li>
  <li>You should be signed up via Selvbetjeningen.</li>
  <li>You will have to decide which project you will work on by August 31 (email: np@math.ku.dk)</li>
  <li>You have to send a proposed title and description of your project by September 3 at 3pm (and you will get feedback).</li>
  <li>You will have to fill out and submit the <a href="https://kunet.ku.dk/fakultet-og-institut/science/uddannelse/projekter/bachelorprojekt/sider/default.aspx">contract</a> before September 4.</li>
</ul>

<p>Below are some additional details on the project.</p>

<h3 id="thesis-contract">Thesis contract</h3>

<p>Use the project descriptions below and take a look at the suggested
literature to come up with a proposed title and description. I will
then read and comment on your proposal and approve it afterwards. Fill
out the <a href="https://kuforms.ku.dk/xform/frontend/FormEngine/v2/ShowForm.aspx?alias=SCI3921&amp;groupId=2&amp;doctype=5&amp;formid=3983">contract
formular</a>,
send the pdf to np@math.ku.dk, and I will submit it. Remember to sign
the contract.</p>

<h3 id="details-on-supervision">Details on supervision</h3>

<p>The type of supervision will depend on whether there are sufficiently
many students working on the same project as you. Overall, you should
expect less supervision with projects on which very few students are
working. So if you select such a project, you should feel comfortable
working independently.</p>

<ul>
  <li>Sufficient students on same project:
    <ul>
      <li>
        <p>During block 1: We will meet in groups and you (the students) will
 present the background literature and theory, and we will have time
 for questions, both general questions as well as questions specific
 to what you are reading.</p>
      </li>
      <li>
        <p>During block 2: We will meet individually. At these meetings you
 are expected to shortly present your progress and have a set of
 prepared questions you would like to discuss.</p>
      </li>
    </ul>
  </li>
  <li>Insufficient students on same project:
    <ul>
      <li>
        <p>During block 1: Short individual meetings. The goal of these
 meetings is to ensure that you spend time on relevant things, so you
 should prepare a detailed plan of what you plan to work on for each
 of these meetings.</p>
      </li>
      <li>
        <p>During block 2: We will meet individually. At these meetings you are
 expected to be shortly present your progress and have a set of
 prepared questions you would like to discuss.</p>
      </li>
    </ul>
  </li>
</ul>

<p>Individual meetings can be onsite or on zoom.</p>

<h3 id="prerequisites">Prerequisites</h3>

<p>The formal prerequisites are the courses <em>Statistical Methods</em> and
<em>Mathematical Statistics</em> (or equivalent), but you are also expected
to be interested in the following:</p>

<ul>
  <li>carry out data analysis and model validation on real data</li>
  <li>implement models and/or data analyses (e.g. by writing R or Python scripts)</li>
  <li>learn to use new software packages and functions</li>
  <li>find relevant literature</li>
  <li>independently read up on the background theory of the project</li>
  <li>write a project that reflects theory as well as applications</li>
</ul>

<h3 id="advice-on-writing">Advice on writing</h3>

<p>You are strongly encouraged to follow the recommendations outlined in
the following <a href="download/writing_guide.pdf">writing guide</a>.</p>

<p>As this is an applied project it is important that your empirical
results can be reproduced. For this it can be helpful to either work
with R Markdown (and perhaps also Tidyverse) or using Jupyter
notebooks, to keep track of the experiments you have performed. See
also:</p>

<ul>
  <li><a href="https://r4ds.had.co.nz">R for Data Science</a> by Garrett Grolemund
and Hadley Wickham</li>
  <li><a href="https://bookdown.org/yihui/rmarkdown/">R Markdown: The Definitive Guide</a> by 
Yihui Xie, J. J. Allaire, Garrett Grolemund</li>
</ul>

<h3 id="more-information-on-formal-requirements">More information on formal requirements</h3>

<ul>
  <li><a href="https://kunet.ku.dk/fakultet-og-institut/science/uddannelse/projekter/bachelorprojekt/sider/default.aspx">Responsibilities and project contract</a></li>
  <li><a href="https://www.science.ku.dk/studerende/studieordninger/faelles_sto/Faelles-del.pdf/">Studieordning</a>, see Bilag 1.</li>
  <li><a href="https://www.science.ku.dk/studerende/studieordninger/bachelor/matematik/sto_matfag_2009.pdf">Studieordning, matematik</a>, see Bilag 3 for the formal thesis objectives.</li>
</ul>

<h2 id="project-suggestions">Project suggestions</h2>

<p>Below we suggest two potential projects. For each data is available
and a rough idea of what you could work on is provided. You are,
however, encouraged to adjust the research question to your own
interests.</p>

<p>If you already have an applied data-based project of your own in mind,
you can send me an email and ask whether it is feasible. Please note
that I may decline to supervise a specific project if it does not fall
within my area of expertise or does not seem sufficiently concrete.</p>

<h3 id="project-1-biomass-prediction---comparing-confidence-intervals">Project 1: Biomass prediction - Comparing confidence intervals</h3>

<p>Quantifying the biomass due to trees is important to assess the stored
carbon in a specific region. However, estimating the entire biomass in
a region is a non-trivial task that often involves counting and
measuring tree sizes. Recent advances in aerial and satellite remote
sensing have provided high-resolution images of regions that together
with deep-learning image analysis can be used to count each tree and
provide its crown area and sometimes even height. Given individual
tree samples for which the biomass as well as crown area and height
has been recorded, one can learn a prediction model to predict biomass
from crown area and height. This model can then be applied to the
remote sensing data to estimate the biomass in large areas.</p>

<p>The prediction models that have been used previously are simple
power-law models that are generally fit using a log-log transformation
in order to simplify the estimation. A further approach is to use
non-linear least squares to directly fit the power-law models,
however, this is often rejected by practioners as non-linear least
squares does not come with the standard model-based prediction
intervals. In this project, you should investigate, based on the tree
biomass data, how the coverage of different types of non-parametric
prediction intervals compare to the currently used log-log transform
based intervals.</p>

<p>The final report should include an overview of the data as well as a
short introduction to any of the methods that were used in the
analysis. The main part of the project should then outline the
discoveries including a clear description of how the different
prediction models were compared. The literature list below provides
tutorials on two approaches to construct non-parametric prediction
intervals: <em>conformal prediction</em> and <em>bootstrap</em>.</p>

<h4 id="data">Data</h4>

<p>For this project you will analyze the data set used in <a href="https://www.sciencedirect.com/science/article/pii/S0378112722006478">Allometric
equations to estimate the dry mass of Sahel woody plants mapped with
very-high resolution satellite
imagery</a>. It
consists of samples of tree foilage, wood and root masses and
corresponding crown area and height.</p>

<h4 id="literature">Literature</h4>

<ul>
  <li><a href="https://arxiv.org/abs/2107.07511">A Gentle Introduction to Conformal Prediction and Distribution-Free Uncertainty Quantification</a></li>
  <li><a href="https://www.sciencedirect.com/science/article/pii/S0169743900001027">The bootstrap: a tutorial</a></li>
</ul>

<h3 id="project-2-human-gut-microbiome---distribution-generalization">Project 2: Human gut microbiome - Distribution generalization</h3>

<p>Microorganisms, such as bacteria, fungi and viruses interact in
various ways with their surroundings. The human body is estimated to
be a habitat for more than 10’000 different microbial species and they
have been associated with various health outcomes such as
cardiovascular disease, metabolic diseases, and obesity. Thanks to
recent advances in gene sequencing technology the field of microbiome
research, which measures the microorganisms based on gene sequencing,
has made significant progress on understanding the links between the
microbiome and its host.</p>

<p>One application of microbiome measurements is to use them as a
diagnostic tool. For this one learns a predictive model that uses the
microbiome to predict a specific phenotype that might be hard to
measure in practice. An important problem with predictive models is
that they may not generalize well to new observations with different
characteristics than in the training sample. In this project, you will
investigate how to construct predictive models based on microbiome
predictors that are robust to distributional shifts. You will work
with the publicily available ‘American Gut Project’ data, which
contains microbiome measurements and corresponding metadata (e.g.,
age, sex, BMI). A potential avenue to investigate might be to consider
different types of variable selection/screening procedures and analyze
how they affect generalization performance.</p>

<p>The final report should include an overview of the data as well as a
short introduction to any of the methods that were used in the
analysis. The main part of the project should then outline the
discoveries including a clear description of how the different
prediction models were compared. The literature list below provides
several tools and methods that can be used in the analysis.</p>

<h4 id="data-1">Data</h4>

<p>For this project you will use data from the American Gut Project
<a href="http://americangut.org/">http://americangut.org</a>. This is an
initiative that enrolls regular people around the world (mainly USA and
UK) and collects data from each participant (consisting of a gut
microbiome sample and a questionnaire).</p>

<h4 id="literature-1">Literature</h4>

<ul>
  <li>
    <p><a href="https://journals.asm.org/doi/10.1128/msystems.00031-18">American Gut: an Open Platform for Citizen Science Microbiome
Research</a>. Paper
describing the American Gut Project.</p>
  </li>
  <li>
    <p><a href="https://github.com/knightlab-analyses/american-gut-analyses">https://github.com/knightlab-analyses/american-gut-analyses</a> Github
repository for American Gut Project, providing details on
downloading full data (not required for this project but may be
useful to get more information on it).</p>
  </li>
</ul>

<p><em>Some tools that might help in the analysis:</em></p>

<ul>
  <li>
    <p><a href="https://arxiv.org/abs/2205.07271">Supervised Learning and Model Analysis with Compositional
Data</a>. A method for a
regression-based analysis that adjust for the compositional nature
of the sequencing data. Method is available as a <a href="https://github.com/shimenghuang/KernelBiome">python
package</a>.</p>
  </li>
  <li>
    <p><a href="https://arxiv.org/abs/1903.01050">Regression models for compositional data: General log-contrast
formulations, proximal optimization, and microbiome data
applications</a>. Extension of the
log-contrast model to include sparsity penalty. Method is available
as a <a href="https://github.com/Leo-Simpson/c-lasso">python package</a>.</p>
  </li>
</ul>

    </div>
  </body>
</html>
